File: spec/factories/motorcycle.rb
FactoryBot.define do
    factory :motorcycle do
      sequence(:title) { |n| "Motorcycle title #{n}" }
      sequence(:description) { |n| "Motorcycle description #{n}" }
      sequence(:image_url) { |n| "Motorcycle image url #{n}"}

      user
    end
  end

File: app/javascript/components/Motorcycle.js
import React from 'react';

const Motorcycle = (props) => {
    const { motorcycle } = props;
    return (
        <div>
            <p>id: {motorcycle.id}</p>
            <p>title: {motorcycle.title}</p>
            <p>description: {motorcycle.description}</p>
            <p>image_url: {motorcycle.image_url}</p>
            <p>user_id: {motorcycle.user.id}</p>
        </div>
    )
}

export default Motorcycle;